extensions/ipynb/src/common.ts

```diff
@@ -40,3 +40,23 @@ export interface CellOutputMetadata {
 	 */
 	__isJson?: boolean;
 }
+
+
+/**
+ * Metadata we store in VS Code cells.
+ * This contains the original metadata from the Jupyuter cells.
+ */
+export interface CellMetadata {
+	/**
+	 * Cell id for notebooks created with the new 4.5 version of nbformat.
+	*/
+	id?: string;
```

Comment:
Brought in a PR that was in jupyter extnesion to support forwards compatibility with the new notebook format 4.5
https://github.com/microsoft/vscode-jupyter/pull/5970

Reply:
This new property isn't required in older versions, and only required in the latest 4.5 version (since we support older versions we're not going to upgrade to 4.5, but ensure users can create notebooks in format 4.2 and open notebooks created using format 4.5)

Note: Assume user creates a notebook in vscode (or older version of jupyter) & then opens this in latest version of Jupyter, the notebook format is upgrade to version 4.5 & jupyter notebook/lab will add the new id attribute.